br/pkg/lightning/common/errors.go

```diff
@@ -215,9 +215,5 @@ func NormalizeOrWrapErr(rfcErr *errors.Error, err error, args ...interface{}) er
 	if IsContextCanceledError(err) {
 		return err
 	}
-	normalizedErr := NormalizeError(err)
```

Comment:
this is not part of the fix
this part of code has bug, it might eat the error stack